instrumentation/github.com/aws/aws-sdk-go/service/otels3/s3.go

```diff
@@ -12,15 +13,14 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3/s3iface"
 	"go.opentelemetry.io/otel/api/global"
```

Comment:
can we remove these `/api/*` since we have the `"go.opentelemetry.io/otel"` imported?

Reply:
they are different packages so we have to import them one by one